build.sbt

```diff
@@ -148,15 +148,16 @@ lazy val jsdocs =
     .settings(
       neverPublish,
       organization                           := "io.indigoengine",
-      libraryDependencies += "org.scala-js" %%% "scalajs-dom" % Dependancies.scalajsDomVersion
+      libraryDependencies += "org.scala-js" %%% "scalajs-dom" % Dependancies.scalajsDomVersion,
+      scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) }
     )
+    .dependsOn(tyrian.js)
+    // .dependsOn(tyrianIndigoBridge.js)
```

Comment:
I'd really like to switch this for `"io.indigoengine" %%% "tyrian" % tyrianDocsVersion` ideally, the current in dev version is the one I want the docs checked against. Feels like that should work but it threw up new problems for me.

Reply:
Ah but I guess that won't work because they're all published against a later version of Scala.js.

Reply:
Yeah, I wouldn't take any of the changes in this PR too seriously :) mainly, it's to demonstrate that things _should_ work and hopefully help track down that mdoc regression in the latest release.